src/applications/claims-status/components/ClosedClaimMessage.jsx

```diff
@@ -63,6 +71,20 @@ export default function ClosedClaimMessage({ claims, onClose }) {
 
       return c;
     });
+};
+
+const getCloseDate = claim => claim.attributes.phaseChangeDate;
```

Comment:
Is the `phaseChangeDate` always the close date? I was working under the assumption that the `phaseChangeDate` is the date the claim changed to _any_ phase.

Reply:
So the way that the vets-api endpoint works currently is there is no attribute to store the `closeDate`. The way that the frontend knows that the claim is closed is by looking at the `open` attribute. It this attribute is set to **false**, then we look at the `phaseChangeDate` to see when the last phase change occurred. The list of claims is filtered down to only include claims where the `open` attribute is set to **false** and where the `phaseChangeDate` is within the last 30 days by the `getRecentlyClosedClaims` method that is called earlier in the component, so in theory `phaseChangeDate` should always represent the close date. The new API will have an explicit property `closeDate` to indicate whether the date the claim was closed on

Reply:
This makes sense, thank you